kubernetes/deployment-staging.tmpl

```diff
@@ -22,15 +22,93 @@ spec:
               value: staging
             - name: REDIS_URL
               value: redis://caesar-staging-redis:6379
-            - name: DOTENV_FILE
-              value: /run/secrets/environment
             - name: PORT
               value: "81"
-          volumeMounts:
-            - name: caesar-staging-environment
-              mountPath: "/run/secrets/environment"
-              subPath: "environment"
-              readOnly: true
+            - name: SECRET_KEY_BASE
+              valueFrom:
+                secretKeyRef:
+                  name: caesar-staging-env-vars
+                  key: SECRET_KEY_BASE
+            - name: RAILS_SERVE_STATIC_FILES
+              valueFrom:
+                secretKeyRef:
+                  name: caesar-staging-env-vars
+                  key: RAILS_SERVE_STATIC_FILES
+            - name: DATABASE_URL
+              valueFrom:
+                secretKeyRef:
+                  name: caesar-staging-env-vars
+                  key: DATABASE_URL
+            - name: AWS_ACCESS_KEY_ID
+              valueFrom:
+                secretKeyRef:
+                  name: caesar-staging-env-vars
+                  key: AWS_ACCESS_KEY_ID
+            - name: AWS_SECRET_ACCESS_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: caesar-staging-env-vars
+                  key: AWS_SECRET_ACCESS_KEY
+            - name: KINESIS_STREAM_PASSWORD
```

Comment:
I don't think these need to be available in the ruby app, happy to remove.

Reply:
we need the aws key to create data exports on s3, but the kinesis one i think we probably don't, unless it's used by the rake task that updates the forwarder

Reply:
I assume the rake task is run outside of our staging / production deployments, maybe locally?

Reply:
I can't see a rake take to update the forwarder... am i missing something?

Reply:
Actually my mistake--it's a python script, not a rake task. And you're right, I doubt that we'd run that from a staging environment rather than a workstation

Reply:
Yeah - docs suggest that the function is declared and uploaded using `lambda-uploader` and the ENV vars are setup in the lambda configuration itself, so outside of this codebase. https://github.com/zooniverse/caesar/blob/master/kinesis-to-http/README.md#deployment